src/fides/api/schemas/privacy_preference.py

```diff
@@ -281,6 +303,12 @@ class CurrentPrivacyPreferenceSchema(TCFAttributes):
     privacy_preference_history_id: Optional[str]
 
 
+class SavePrivacyPreferencesResponse(FidesSchema):
+    """Response schema when saving privacy preferences"""
+
+    preferences: List[CurrentPrivacyPreferenceSchema]
```

Comment:
hmmm @allisonking wanted to get your opinion here.  I forgot this key has meaning in the request - that the "preferences" key is for saving preferences against privacy notices.  Would it make more sense to break the response up further, "preferences", "purpose_consent_preferences", "purpose_legitimate_interests_preferences", etc?

Reply:
discussed in slack, we did up breaking the response up to mirror the request!